pkg/docker/config/config.go

```diff
@@ -83,7 +90,12 @@ func SetCredentials(sys *types.SystemContext, registry, username, password strin
 		// External helpers.
```

Comment:
Above, the `CredHelpers[registry]` case also needs to be rejected. `RemoveAuthentication` also needs to handle that case.

Reply:
Added an additional path check for both cases.